Coveralls.psm1

```diff
@@ -150,7 +152,13 @@ function Format-Coverage {
     )
 
     $fileCoverageArray = @()
-    $pesterResults = Invoke-Pester -CodeCoverage $Include -Quiet -PassThru
+    if (!$pesterResults) {
+        $pesterResults = Invoke-Pester -CodeCoverage $Include -Quiet -PassThru
+    }
+    if (!$pesterResults.CodeCoverage) {
+        Write-Error 'Please provide pester results with code coverage using the -CodeCoverage parameter'
+        return;
```

Comment:
Opted for `return` here since `Write-Error` can be made fatal by setting `$ErrorActionPreference = 'Stop'`. Thoughts?